libbeat/outputs/mode/failover.go

```diff
@@ -183,6 +183,7 @@ func (f *FailOverConnectionMode) publish(
 		}
 		if f.maxAttempts > 0 && fails == f.maxAttempts {
 			// max number of attempts reached
+			messagesDropped.Add(1)
```

Comment:
can be moved to line 193. Having this close to SignalFailed (plus put in another method) better clarifies these to belonging together.